### DISCUSSION TITLE
pcp_combine with python embedding

### DISCUSSION THREAD

**Discussion by davis-wesley-12345** [dtcenter/METplus#1300] [Use Cases] [answered]
Are there any examples that I can refer to when trying to use pcp_combine with python embedding? Asking this as a follow up to a question posted in today's training.  Thank you!

**Comment by georgemccabe** [comment, marked as answer]
Hi @davis-wesley-12345,

Here is the documentation for the METplus use case example that runs PCPCombine using Python Embedding:

https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/PCPCombine/PCPCombine_python_embedding.html#sphx-glr-generated-met-tool-wrapper-pcpcombine-pcpcombine-python-embedding-py

The use case generates a command that looks something like this:

`/usr/local/met/bin/pcp_combine -v 2 -name APCP -add PYTHON_NUMPY 'name="/METplus-4.0.0/parm/use_cases/met_tool_wrapper/PCPCombine/sum_IMERG_V06_HDF5.py /d1/METplus_Data/met_test/new/imerg IRprecipitation 2018010213 02";' /d1/out/met_tool_wrapper/PCPCombine/PCPCombine_combine_py_embed/IMERG.20180102_1300_A06h`

The Python script used to read data is found in the value of the "name" attribute followed by the arguments to the script. Instead of a path to an input file, the keyword "PYTHON_NUMPY" is supplied as an argument. Please note that this specific use case requires additional Python packages besides the minimum requirements for Python Embedding in MET. This is described in the use case documentation page.

Please let me know if you have any other questions.
